rust/pact_models/src/sync_interaction.rs

```diff
@@ -142,20 +143,20 @@ impl RequestResponseInteraction {
     };
     let provider_states = ProviderState::from_json(pact_json);
     let request = match pact_json.get("request") {
-      Some(v) => Request::from_json(v, spec_version),
+      Some(v) => Request::from_json(v, spec_version)?,
       None => Request::default()
     };
     let response = match pact_json.get("response") {
-      Some(v) => Response::from_json(v, spec_version),
+      Some(v) => Response::from_json(v, spec_version)?,
       None => Response::default()
     };
-    RequestResponseInteraction {
+    Ok(RequestResponseInteraction {
       id,
       description,
       provider_states,
       request,
-      response
-    }
+      response,
```

Comment:
Commas?

Reply:
(same situation as the other trailing-commas discussion)